src/library/Uno.Material/MaterialResourcesV2.cs

```diff
@@ -77,66 +69,66 @@ private IEnumerable<(string ResourceKey, string SharedKey, bool IsDefaultStyle)>
 		{
 			var result = new List<(string ResourceKey, string SharedKey, bool IsDefaultStyle)>();
 
-			Add("M3MaterialCheckBoxStyle", isImplicit: true);
-			Add("M3MaterialAppBarButtonStyle", isImplicit: true);
-			Add("M3MaterialCommandBarStyle", isImplicit: true);
-			Add("M3MaterialRadioButtonStyle", isImplicit: true);
-			Add("M3MaterialDisplayLarge");
-			Add("M3MaterialDisplayMedium");
-			Add("M3MaterialDisplaySmall");
-			Add("M3MaterialHeadlineLarge");
-			Add("M3MaterialHeadlineMedium");
-			Add("M3MaterialHeadlineSmall");
-			Add("M3MaterialTitleLarge");
-			Add("M3MaterialTitleMedium");
-			Add("M3MaterialTitleSmall");
-			Add("M3MaterialLabelLarge");
-			Add("M3MaterialLabelMedium");
-			Add("M3MaterialLabelSmall");
-			Add("M3MaterialBodyLarge");
-			Add("M3MaterialBodyMedium", isImplicit: true);
-			Add("M3MaterialBodySmall");
-			Add("M3MaterialOutlinedTextBoxStyle");
-			Add("M3MaterialFilledTextBoxStyle", isImplicit: true);
-			Add("M3MaterialOutlinedPasswordBoxStyle");
-			Add("M3MaterialFilledPasswordBoxStyle", isImplicit: true);
-			Add("M3MaterialElevatedButtonStyle");
-			Add("M3MaterialFilledButtonStyle", isImplicit: true);
-			Add("M3MaterialFilledTonalButtonStyle");
-			Add("M3MaterialOutlinedButtonStyle");
-			Add("M3MaterialTextButtonStyle");
-			Add("M3MaterialIconButtonStyle");
-			Add("M3MaterialCalendarViewStyle", isImplicit: true);
-			Add("M3MaterialCalendarDatePickerStyle", isImplicit: true);
-			Add("M3MaterialFlyoutPresenterStyle", isImplicit: true);
-			Add("M3MaterialMenuFlyoutPresenterStyle", isImplicit: true);
-			Add("M3MaterialNavigationViewStyle", isImplicit: true);
-			Add("M3MaterialNavigationViewItemStyle", isImplicit: true);
-			Add("M3MaterialListViewStyle", isImplicit: true);
-			Add("M3MaterialListViewItemStyle", isImplicit: true);
-			Add("M3MaterialTextToggleButtonStyle", isImplicit: true);
-			Add("M3MaterialIconToggleButtonStyle");
-			Add("M3MaterialDatePickerStyle", isImplicit: true);
-			Add("M3MaterialComboBoxStyle", isImplicit: true);
-			Add("M3MaterialFabStyle");
-			Add("M3MaterialSurfaceFabStyle");
-			Add("M3MaterialSecondaryFabStyle");
-			Add("M3MaterialTertiaryFabStyle");
-			Add("M3MaterialSmallFabStyle");
-			Add("M3MaterialSurfaceSmallFabStyle");
-			Add("M3MaterialSecondarySmallFabStyle");
-			Add("M3MaterialTertiarySmallFabStyle");
-			Add("M3MaterialLargeFabStyle");
-			Add("M3MaterialSurfaceLargeFabStyle");
-			Add("M3MaterialSecondaryLargeFabStyle");
-			Add("M3MaterialTertiaryLargeFabStyle");
-			Add("M3MaterialExtendedFabStyle");
-			Add("M3MaterialSurfaceExtendedFabStyle");	
-			Add("M3MaterialSecondaryExtendedFabStyle");	
-			Add("M3MaterialTertiaryExtendedFabStyle");
-			Add("M3MaterialSliderStyle");
-			Add("M3MaterialHyperlinkButtonStyle", isImplicit: true);
-			Add("M3MaterialSecondaryHyperlinkButtonStyle");
+			Add("MaterialCheckBoxStyle", isImplicit: true);
```

Comment:
Is the related doc for the matrix style needs also some update ? Missing styles or implicit column?

Reply:
you can ran the script again; it will parse this method && all styles in xaml
to do a join